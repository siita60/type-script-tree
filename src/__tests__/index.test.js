const { TestWatcher } = require('jest');
const { main } = require('../');

// 空のディレクトリ確認
describe('toy-tree', () => {
  let stdout;
  let stderr;
  const exec = (argv) => main(argv, stdout, stderr);

  beforeEach(() => {
    stdout = jest.fn();
    stderr = jest.fn();
  });

  test('fixtures', () => {
    const code = exec(['fixtures']);
    expect(stdout.mock.calls[0][0]).toMatchSnapshot();
    expect(code).toBe(0);
  });
});
