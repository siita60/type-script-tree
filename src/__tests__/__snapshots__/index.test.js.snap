// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`toy-tree fixtures 1`] = `
"fixtures
|-- [36mdir1[39m
|  |-- file1.txt
|  |-- file2.txt
|  \`-- [36mnest1[39m
|   |-- deep1.txt
|   \`-- deep2.txt
|-- [36mdir2[39m
|  |-- file1.txt
|  |-- file2.txt
|  \`-- file3.txt
|-- file1.txt
\`-- file2.txt
"
`;
